ui/src/main/js/page/certificates/EditCertificateCell.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { createUseStyles } from 'react-jss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import CertificateModal from 'page/certificates/CertificateModal';
+
+const useStyles = createUseStyles({
+    editCell: {
+        '&:hover': {
+            cursor: 'pointer',
+        }
+    }
+});
+
+const EditCertificateCell = ({ data }) => {
```

Comment:
Define PropTypes for validation.

Reply:
Addressed in my most recent commit, 43792b148adc1a01681d6bcca568e71d6bc31d8